franklin/core/urls.py

```diff
@@ -1,18 +1,16 @@
 from django.conf.urls import url
 
-from .views import dashboard_init, health
+from .views import health
 from builder.views import UpdateBuildStatus
 from github.views import deploy_hook, deployable_repos, repository_detail, \
         repository_list, get_auth_token
 
 urlpatterns = [
     url(r'^auth/github/$', get_auth_token, name='get_token'),
-    url(r'^dashboard/init/$', dashboard_init, name='dashboard_init'),
```

Comment:
@camiriveron - Instead, you will make a GET call to `auth/github/`. You will still POST to `auth/github` as before.